src/results/electric_tariff.jl

```diff
@@ -14,7 +14,38 @@
 - `year_one_export_benefit_before_tax` export credits over the first year, before considering tax benefits. A positive value indicates a benefit. 
 - `lifecycle_coincident_peak_cost_after_tax` lifecycle coincident peak charge in present value, after tax
 - `year_one_coincident_peak_cost_before_tax` coincident peak charge over the first year
+
+Outputs related to electric tariff (year-one rates and costs; not escalated):
+- `monthly_fixed_cost` the fixed monthly cost of electricity for each month per chosen electric tariff
+- `energy_rate_series` dictionary for cost of electricity, each key corresponds to a tier with value being \$/kWh timeseries
+- `energy_rate_tier_limits` dictionary for cost of electricity, each key corresponds to a tier with value being \$/kWh timeseries
+- `energy_rate_average_series` average energy rate across all tiers as \$/kWh timeseries
+- `facility_demand_monthly_rate_series` facility demand charges (rows = months, columns = # of tiers)
+- `facility_demand_monthly_rate_tier_limits` facility demand charges (rows = months, columns = # of tiers)
+- `tou_demand_rate_series` is a dictionary with TOU demand charges as timeseries for each timestep
+- `demand_rate_average_series` average TOU demand rate across all tiers as \$/kW timeseries
+- `tou_demand_rate_tier_limits`
+
+Outputs related to eventual costs of electricity (year-one rates and costs; not escalated):
+- `energy_cost_series_before_tax` timeseries of cost of electricity purchases from the grid (grid to total net load)
+- `monthly_energy_cost_series_before_tax`
+- `monthly_facility_demand_cost_series_before_tax`
+- `tou_demand_metrics` -> month: Month this TOU period applies to
```

Comment:
why does `tou_demand_metrics` appear so many times here? And I'm a bit confused about why some of $/kW metrics would appear in this section vs the section above. It seems like that section is intended to be rates and this one should be total costs?